common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/responses/ApiErrorResult.kt

```diff
@@ -30,6 +30,7 @@ import com.microsoft.identity.common.java.logging.DiagnosticContext
 open class ApiErrorResult(
     open val error: String?,
     open val errorDescription: String?,
-    open val correlationId: String,
+    open val details: List<Map<String, String>>? = null,
+    open val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId,
```

Comment:
I think we already mentioned that you should grab it from eSTS if available

Reply:
I replied in the different comment thread about correlation IDs